constructor/nsis/_nsis.py

```diff
@@ -229,6 +230,18 @@ def rm_regkeys():
             pass
 
 
+def win_del(dirname):
+    try:
+        out = check_output('DEL /F/Q/S {} > NUL'.format(dirname), shell=True, stderr=STDOUT, cwd=dirname)
```

Comment:
Wouldn't it be easier to provide the arguments for `DEL` as list? For example:
```suggestion
        out = check_output(['DEL', '/F', '/Q', '/S', dirname], shell=True, stdout=subprocess.DEVNULL, stderr=STDOUT, cwd=dirname)
```

Reply:
You may know more about this than me, so take this with a grain of salt.  My experience is that for things that are shell functions, not actual programs (which is what DEL is here), you must use shell=True, and you must make a single string, not a list of strings.

Reply:
After your comment I wasn't sure any more, but I just tested it and if `shell=True`, you can split the arguments to shell commands like `dir` or `del`. So something like `check_output(['del', some_dir], shell=True)` should work.
The solution with a single string should also work, however I always fear issues with broken quoting of arguments with whitespace, so I always prefer argument lists instead of strings.